cmd/yurt-controller-manager/controller-manager.go

```diff
@@ -26,10 +26,8 @@ import (
 	"time"
 
 	"k8s.io/component-base/logs"
-
 	// load all the prometheus client-go plugin
```

Comment:
The golangci-lint error can be find here.
It seems that we should retain the empty line above the annotated import package.

Reply:
thank you.

i removed the empty lines because my local golangci-lint (the latest version) throw errors for them.